src/test/kotlin/bsu/cc/schedule/Utils.kt

```diff
@@ -0,0 +1,9 @@
+package bsu.cc.schedule
+
+/**
+ * list equality depends on position, so sort sub-lists first
+ * map returns list, so convert back to set
+ * sets equality does not depend on position
+ */
+fun checkOverlapsAreEqual(actual: Set<List<ClassSchedule>>, expected: Set<List<ClassSchedule>>): Boolean
```

Comment:
Are we going to have need of this outside of testing? If not, perhaps it should just be in the test file.

Reply:
This is in test file path. I'm pretty sure I'll need more utility functions and I decided to place them in their own file instead of potentially copying them among files. I'll move it into the test file if no other tests end up needing it.

Reply:
Good news! I moved a test file into this package which had a duplicate utility function. I move said function to the utils file. It now has a purpose!